haiku-apps/passwordgenerator/passwordgenerator-1.0.0.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A simple password generator app"
+DESCRIPTION="This simple password generator app was originally created on pascal." 
+HOMEPAGE="https://example.com"
+COPYRIGHT="2022 LinuxUser"
+LICENSE="MIT" 
+REVISION="1"
+srcGitRev="54855ece8f2b9d5c9603774a78132b4d7e7fd2fa" 
+SOURCE_URI="https://github.com/HaikuUserLinux/PasswordGeneratorForHaiku/archive/$srcGitRev.tar.gz" 
+CHECKSUM_SHA256="40fbbc8e3253259b4dee9da667973ecf48b7607f124cac9a06fd3a0d58c5fcc1" 
+SOURCE_DIR="PasswordGeneratorForHaiku-$srcGitRev"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+addfix=haiku
+devpref=_devel
+PROVIDES="
+	passwordgenerator$secondaryArchSuffix
+	app:passwordgenerator$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libQt6Core$secondaryArchSuffix
+	lib:libQt6Gui$secondaryArchSuffix
+	lib:libQt6Widgets$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	$addfix$secondaryArchSuffix$devpref
+	devel:libQt6Core$secondaryArchSuffix
+	devel:libQt6Gui$secondaryArchSuffix
+	devel:libQt6Widgets$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:qmake$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	qmake . INSTALL_ROOT=$prefix
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir $appsDir
+	cp passwordgenerator $appsDir
```

Comment:
I don't think it's a good idea that the binary's name in $appsDir is different than in Deskbar.
Either "passwordgenerator" or "Password Generator".
Personally, I'd go with something shorter anyway, e.g. "PassGenerator", "PassGen" or something whimsical like "Passwordler".